docs/user-guide.md

```diff
@@ -16,7 +32,154 @@
+
+![Stop command](/docs/user-guide-screenshots/devModeStop.png)
+
+## Debug your application
+
+To attach the debugger, you must have a [running server](#run-your-application-on-liberty-using-dev-mode). Once running, click on the `Attach debugger` command or select the `Liberty: Attach debugger` command in the command palette followed by your desired application.
+
+![Attach Debugger](/docs/user-guide-screenshots/attach_debugger.png)
+
+When attached, Visual Studio Code's [debug options](https://code.visualstudio.com/docs/editor/debugging) will become available.
+
+![Debug Mode](/docs/user-guide-screenshots/debugger_mode.png)
+
+## Manually add your Liberty project to the dashboard
+
+In the event that your Liberty project is not automatically detected by the Liberty dashboard, you can manually add your Liberty project to the Liberty dashboard. To manually add your Liberty project to the in the Liberty dashboard, right click into an empty space in the project explorer and select **Add project to Liberty Dashboard** or select the `Liberty: Add project to the tool window` command in the command palette followed by your desired application.
+
+![Liberty add project to tool window](/docs/user-guide-screenshots/add_liberty_project.png)
+
+You are prompted with a list of projects that are not already displayed in the in the Liberty dashboard.
+
+![Liberty add project to tool window selection](/docs/user-guide-screenshots/add_liberty_project_2.png)
+
+You can remove manually added Liberty projects from the in the Liberty dashboard right clicking into an empty space in the project explorer and selecting `Remove project from Liberty Dashboard` or by selecting the `Liberty: Remove project from the tool window` command in the command palette followed by your desired application.
+
+![Liberty Remove Project](/docs/user-guide-screenshots/removeLibertyProject.png)
+
+## Configure a Liberty server
+
+Liberty configuration assistance [(code completion, diagnostics and quick-fixes)](https://github.com/OpenLiberty/liberty-language-server#features) provides editing assistance in Liberty `server.xml`, `server.env`, and `bootstrap.properties` files.
+
+1. Start the project in dev mode by using one of the previously described Liberty dashboard or command palette commands. Dev mode installs the Liberty features that are required for your application.
+2. Open any of the supported Liberty configuration files.
+3. To use Liberty-specific code completion, press <kbd>Ctrl/Cmd</kbd> + <kbd>Space</kbd> anywhere within the document. A drop-down list of completion suggestions appears.
+
+Liberty configuration assistance is offered through the Liberty Config Language Server. For more information, see the [project documentation in GitHub](https://github.com/OpenLiberty/liberty-language-server#liberty-config-language-server).
+
+![LCLS server.xml completion](/docs/user-guide-screenshots/serverXmlCompletion.png)
+
+## Develop with Jakarta EE and MicroProfile APIs
```

Comment:
This section should mirror the structure of https://github.com/OpenLiberty/liberty-tools-intellij/blob/main/docs/user-guide.md#develop-with-jakarta-ee-and-microprofile-apis